src/api/custom-resolvers/queries/get-ordnance-survey-addresses/index.ts

```diff
@@ -36,6 +36,17 @@ const getOrdnanceSurveyAddresses = async (root: any, variables: OrdnanceSurveyVa
 
     const response = await ordnanceSurvey.get(postcode);
 
+    /**
+     * if received status is 400
+     * then return noAddressesFound flag
```

Comment:
Not a blocker, could do in another PR, to be aligned with the other comments
```suggestion
     * return success=false and noAddressesFound=true
```